PayPal/src/main/java/com/braintreepayments/api/PayPalClient.java

```diff
@@ -43,14 +43,14 @@ private boolean browserSwitchNotPossible(FragmentActivity activity) {
 
     private static Exception createPayPalError() {
         return new BraintreeException("PayPal is not enabled. " +
-                "See https://developers.braintreepayments.com/guides/paypal/overview/android/ " +
+                "See https://developer.paypal.com/braintree/docs/guides/paypal/overview/android/v4 " +
```

Comment:
I think in the past we've had discussions around whether or not changing an error message is a breaking change - I think it's better to change than to have broken links, but wanted to double check.

Reply:
We reverted the changes here and in the 3DS client and added next major version notes to update them (at least).